module/proto/gravity/v1/query.proto

```diff
@@ -203,8 +208,13 @@ message QueryDenomToERC20Response {
 }
 
 message QueryAttestationsRequest {
```

Comment:
@facundomedica How would you like to handle `get by claim type`? Should `type` be a string here?